packages/serverpod/lib/src/database/migrations/migration_manager.dart

```diff
@@ -213,7 +214,7 @@ class MigrationManager {
 
       sqlToExecute.add((version: latestVersion, sql: sqlDefinition));
     } else {
-      var newerVersions = _getVersionsToApply(fromVersion);
+      var newerVersions = _getVersionsToApply(fromVersion!);
```

Comment:
Chore: Revert changes in this file.

Here is another example of when ! is now required because the analyser cannot deduce that `fromVersion` can't be null at this point because we don't utilize `fromVersion == null`.